.terraform/main.tf

```diff
@@ -0,0 +1,73 @@
+terraform {
+  required_version = ">= 0.14.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.12.1"
+    }
+  }
+}
+
+provider "aws" {
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+  region     = var.aws_region
+}
+
+#========================================================
+# VPC
+#========================================================
+resource "aws_vpc" "vpc" {
+  cidr_block = var.vpc_cidr
+
+  tags = {
+    Name = var.vpc_tag_name
+  }
+}
+
+resource "aws_subnet" "vpc_public_subnet" {
+  vpc_id = aws_vpc.vpc.id
+
+  cidr_block = var.vpc_subnet_cidr
+
+  tags = {
+    Name = var.vpc_subnet_tag_name
+  }
+}
+
+#========================================================
+# EC2
+#========================================================
+resource "aws_security_group" "security_group" {
+    vpc_id = aws_vpc.vpc.id
+
```

Comment:
I would give this a name such as Office VPN Access or separate the HTTP and SSH access into separate security groups.